### PR TITLE
Space Saver Hub Project 

### PR DESCRIPTION
**Pull request by coker2019**
-Config & basic setup.

-Redux: Fetch data and update redux store.

-Render UI: lists.

-Write actions and reducers for canceling rockets/dragons and leaving missions.

- Add test to components.

### CODE REVIEW

File: src/_test_/RocketItem.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      .create(
+        <Provider store={store}>
+          <RocketItem />
+        </Provider>,
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (ghost) [inline src/_test_/RocketItem.test.js:20]: - [ ] 🎉 Great job on thoroughly testing for proper rendering! 👍 Don't forget the importance of testing your pure functions too. 🧪 Ensuring your app behaves as expected is the key! Keep up the awesome work in this PR! 🚀

File: src/redux/Rocket/rocketSlice.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+// fetch books from the books API
+export const fetchRockets = createAsyncThunk('rockets/fetchRockes', async (_, { rejectWithValue }) => {
+  try {
+    const response = await axios.get(url);
+    return response.data;
+  } catch (error) {
+    return rejectWithValue('something went wrong');
+  }
+});
```
Comment (ghost) [inline src/redux/Rocket/rocketSlice.js:20]: - [ ] I've observed that when I book a rocket 🚀 and then navigate to the profile page 🧑‍🚀, the booking is correctly added there. However, upon returning to the rockets page 🪐, the booking disappears, which is not the expected behavior. 🤔 Please refer to your missions implementation to achieve consistent behavior for rocket bookings. Keep up the good work! 💪🌟

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights


✅ Great work using  React Testing Library to test proper rendering👍 
✅ Good work setting up redux correctly👍



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @AlyAtalla and @coker2019, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**